### PR TITLE
Feature ram

### PR DESCRIPTION
**Pull request by AutoViML**

### CODE REVIEW

File: auto_ts/__init__.py [hunk 1]
```diff
@@ -71,22 +44,29 @@ def warn(*args, **kwargs):
                    time_series_plot, print_static_rmse, print_dynamic_rmse
 
 
-class AutoTimeSeries:
+class auto_timeseries:
```
Comment (ngupta23) [inline auto_ts/__init__.py:47]: Hi @rsesha , [PEP8 recommendation](https://www.python.org/dev/peps/pep-0008/#class-names) is to use Camel Case for class names.

File: auto_ts/__init__.py [hunk 2]
```diff
@@ -855,41 +750,50 @@ def get_ml_dict(self):
     def predict(
         self,
         model: str = 'best',
-        X_exogen: Optional[pd.DataFrame]=None,
+        testdata: Optional[pd.DataFrame]=None,
         forecast_period: Optional[int] = None,
-        simple: bool = True) -> Optional[np.array]:
```
Comment (ngupta23) [inline auto_ts/__init__.py:860]: Not all models return the confidence intervals. Should the default be left as False?

File: auto_ts/models/build_prophet.py [hunk 3]
```diff
@@ -110,54 +111,68 @@ def fit(self, ts_df: pd.DataFrame, target_col: str, cv: Optional[int], time_col:
...
             print(f"Horizon Start: {dft.iloc[-self.forecast_period]['ds']}")
 
-        #horizon_days = (dft['ds'].max() - dft.iloc[-forecast_start]['ds']).days 
-        horizon_days = (dft['ds'].max() - dft.iloc[-(self.forecast_period+1)]['ds']).days 
-        
-        initial_days = total_days - NFOLDS * horizon_days
-        period_days = horizon_days
+        #horizon_days = (dft['ds'].max() - dft.iloc[-forecast_start]['ds']).days
+        #horizon_days = min(365, (dft['ds'].max() - dft.iloc[-(self.forecast_period+1)]['ds']).days )
+        horizon_days = int(total_days/3)
```
Comment (ngupta23) [inline auto_ts/models/build_prophet.py:129]: Hi @rsesha , I am not following the logic here for these computations. Can you add some comments on what this is supposed to do? Thanks!

**Review by ngupta23 — COMMENTED** [review]
Hi @rsesha , I compleetd my review. Almost everything looks good. Just some minor comments.

Just wanted to check with you if you reran the unit tests. I did not see any changes to those files and the change in argument names, class names and default values will definitely break them. I would recommend updating the unit tests and rerunning them to ensure that they are not breaking. Then we can merge to develop.

Thanks!

**Review by rsesha — APPROVED** [review]
Incorporated comments and changes. approved.

File: auto_ts/__init__.py (hunk 1 above)
Comment (ngupta23) [reply auto_ts/__init__.py:47]: So just to confirm, you prefer to keep it lower case?

File: auto_ts/models/build_prophet.py (hunk 3 above)
Comment (ngupta23) [reply auto_ts/models/build_prophet.py:129]: Hi @rsesha , Do you want to add some notes here on the logic so its easy to follow?

**Review by ngupta23 — APPROVED** [review]

**Pull request merged**